### DISCUSSION TITLE
Is there a way to find hard training sample from unlabeled ASR data

### DISCUSSION THREAD

**Discussion by mehadi92** [NVIDIA-NeMo/NeMo#7654] [Q&A] [answered]
Hi, 
I have 17K hours of unlabeled WAV data. Is there any way to find hard training sample from it?

Thanks

**Comment by egorsmkv** [comment]
You can measure the noise in this data, for example

**Reply by mehadi92** [reply]
Hi, @egorsmkv, 

Do you mean by calculating SNR for each data sample?

**Reply by egorsmkv** [reply, marked as answer]
Yes, to measure SNR
